horace_core/sqw/file_io/sqw_fopen.m

```diff
@@ -9,6 +9,16 @@
 % fh          -- handle of the file open for binary access with format
 %                common to Horace sqw file
 % Throws if open operation is unsuccessful
+[fp,fn] = fileparts(file);
+if ~isfolder(fp)
```

Comment:
This looks like you're changing the behaviour to dump to folders rather than just files. Where is this documented? 

I thought `tmp_dir` always created the folder

Reply:
apparently not. The change was caused by absence of tmp_dir 

There are no global change in behaviour. If folder is not there, file created together with folder instead of failing. It looks like a good thing to have and what is expected. No reason prohibiting it or expecting something else.